### PR TITLE
Updated sample, bumped Python client version [API-1467]

### PR DESCRIPTION
**Pull request by yuce**
* Updated code sample according to the the unified Cloud client sample.
* Bumped the Python client version.

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -1,95 +1,130 @@
...
+    cities.put("6", city("Turkey", "Istanbul", 15_636_243))
+    cities.put("7", city("Brazil", "Sao Paulo", 22_429_800))
+    cities.put("8", city("Brazil", "Rio de Janeiro", 13_634_274))
+    map_size = cities.size()
+    print(f"'cities' map now contains {map_size} entries.")
+    print("-"*20)
+
+
+def sql_example(client: hazelcast.HazelcastClient):
+    svc = client.sql
```
Comment (mdumandag) [inline client.py]: Can we rename `svc` to `sql_service` to be consistent with the parameter names?

File: client.py [hunk 2]
```diff
@@ -1,95 +1,130 @@
...
+def create_mapping_for_capitals(sql_service: hazelcast.client.SqlService):
     print("Creating a mapping...")
-    # See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
-    mapping_query = "CREATE OR REPLACE MAPPING cities TYPE IMap " \
-                    "OPTIONS ('keyFormat'='varchar','valueFormat'='varchar')"
-    hz_client.sql.execute(mapping_query).result()
-    print("The mapping has been created successfully.")
-    print("--------------------")
+    # See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
+    mappingQuery = '''
```
Comment (mdumandag) [inline client.py]: should be snake case

File: client.py [hunk 3]
```diff
@@ -1,95 +1,130 @@
...
-    ('Turkey','Ankara'),
-    ('United States','Washington, DC');
-    """
-    hz_client.sql.execute(insert_query).result()
-    print("The data has been inserted successfully.")
-    print("--------------------")
 
+def populate_capitals(sql_service: hazelcast.client.SqlService):
+    print("Inserting data via SQL...")
+    insertQuery = '''
```
Comment (mdumandag) [inline client.py]: should be snake case

File: client.py [hunk 4]
```diff
@@ -1,95 +1,130 @@
...
+        country = row.get_object("__key")
+        city = row.get_object("__key")
+        print(f"Country name: {country}; Capital name: {city}")
+    print("-" * 20)
 
-    select_cities_by_country(hz_client, "AU")
 
-    select_countries_and_cities(hz_client)
+def json_serialization_example(client):
+    svc = client.sql
```
Comment (mdumandag) [inline client.py]: same as above

File: client.py [hunk 5]
```diff
@@ -153,74 +180,74 @@ def create_mapping_for_cities(hz_client):
...
         print("%4s | %15s | %20s | %15s |" %
               (row["isoCode"], row["country"], row["city"], row["population"]))
-    print("-----------------------------------------------------------------")
+    print("-" * 20)
+
+
+def nonstop_map_example(client: hazelcast.HazelcastClient):
+    print("Now the map named 'map' will be filled with random entries.")
+    map = client.get_map("map").blocking()
+    iterationCounter = 0
```
Comment (mdumandag) [inline client.py]: snake case

File: client.py [hunk 6]
```diff
@@ -153,74 +180,74 @@ def create_mapping_for_cities(hz_client):
...
+            iterationCounter = 0
+            print(f"Current map size: {map.size()}")
 
 
 logging.basicConfig(level=logging.INFO)
 HazelcastCloudDiscovery._CLOUD_URL_BASE = "YOUR_DISCOVERY_URL"
 client = hazelcast.HazelcastClient(
-    cluster_name="YOUR_CLUSTER_NAME",
-    cloud_discovery_token="YOUR_CLUSTER_DISCOVERY_TOKEN",
+    cluster_name="pr-3522",
```
Comment (mdumandag) [inline client.py]: Some leftovers from testing

**Review by mdumandag — COMMENTED** [review]
I just reviewed the client.py but the same comments applies to client_with_ssl as well

File: client.py (hunk 1 above)
Comment (yuce) [reply client.py]: Updated at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

File: client.py (hunk 2 above)
Comment (yuce) [reply client.py]: Fixed at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

File: client.py (hunk 3 above)
Comment (yuce) [reply client.py]: Fixed at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

File: client.py (hunk 5 above)
Comment (yuce) [reply client.py]: Fixed at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

File: client.py (hunk 4 above)
Comment (yuce) [reply client.py]: Updated at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

File: client.py (hunk 6 above)
Comment (yuce) [reply client.py]: Removed them at: https://github.com/hazelcast/hazelcast-cloud-python-sample-client/pull/9/commits/62b8e5462a629de929e28272260d2ea6365239c4

**Review by mdumandag — APPROVED** [review]

**Pull request merged**